draft-ietf-ppm-dap.md

```diff
@@ -751,7 +751,13 @@ To get the aggregate of a batch, the Collector issues a query specifying the
 batch ID of interest (see {{query}}). The Collector may not know which batch ID
 it is interested in; in this case, it can also issue a query of type
 `current_batch`, which allows the Leader to select a recent batch to aggregate.
-The Leader SHOULD select a batch which has not yet began collection.
+The Leader MUST select a batch which has not yet began collection.
+
```

Comment:
Why RECOMMEND and not MUST? What are you trying to get implementations to do here?

Reply:
I see. My gut feeling is that this paragraph is not really necessary: since a collection job is a resource associated with an ID, it seems obvious that you would have to remember that ID in order to complete the job later on.

Am I missing anything? Perhaps you encountered a bug in Janus that made you feel this text was needed?